docker/demo_server/docker-compose.yml

```diff
@@ -13,9 +13,11 @@ services:
       WEB_HOST: '${WEB_URL}'
       JWT_SECRET: '${JWT_SECRET}'
     volumes:
-      - ./app_postgres_on_port_${PORT_APP}:/var/lib/postgresql/12/main/
-      - ./app_snapshots_on_port_${PORT_APP}:/usr/src/application-manager-server/build/database/_snapshots
-      - ./app_file_on_port_${PORT_APP}:/usr/src/application-manager-server/build/files
+      - ./conforma_on_port_${PORT_APP}/postgres:/var/lib/postgresql/12/main/
+      - ./conforma_on_port_${PORT_APP}/snapshots:/usr/src/application-manager-server/build/database/_snapshots
+      - ./conforma_on_port_${PORT_APP}/files:/usr/src/application-manager-server/build/files
+      - ./conforma_on_port_${PORT_APP}/localisation:/usr/src/application-manager-server/build/localisations
+      - ./conforma_on_port_${PORT_APP}/preferences:/usr/src/application-manager-server/build/preferences
```

Comment:
@andreievg -- just checking with you that this folder structure will be okay for volumes? 

Still separate volumes for each folder, but grouped by installation to make it a bit tidier.